R/tm_g_association.R

```diff
@@ -1,27 +1,31 @@
-#' Stack Plots of variables and show association with reference variable
-#' @md
+#' Stack plots of variables and show association with reference variable
+#'
+#' Module provides functionality for visualizing the distribution of variables and
+#' their association with a reference variable.
+#' It supports configuring the appearance of the plots, including themes and whether to show associations.
+#'
+#'
+#' @note For more examples, please see the vignette "Using association plot" via
+#' `vignette("using-association-plot", package = "teal.modules.general")`.
 #'
 #' @inheritParams teal::module
 #' @inheritParams shared_params
 #' @param ref (`data_extract_spec` or `list` of multiple `data_extract_spec`)
-#'   reference variable, must set `multiple = FALSE`.
+#' Reference variable, must set `multiple = FALSE`.
```

Comment:
```suggestion
#' Reference variable. It can only have single selection options in the `select_spec` by setting `multiple = FALSE`.
```

The part `must set 'multiple = FALSE'` was hard to understand for me. A little more context would be helpful. What do you think about something like this?

Reply:
this would be more clear .WDYT

`#' Reference variable, must set `multiple = FALSE` to ensure each `data_extract_spec` specifies only one variable, preventing multiple selections within a single specification.`

Reply:
`multiple` is the argument of `select_spec()` and not `data_extract_spec` so not mentioning about `select_spec` might confuse people.

Reply:
> #' Reference variable, must accepts a `data_extract_spec` with `select_spec(multiple = FALSE) `to ensure single selection option.

 how about this ?

Reply:
I like this one 👍🏽 It's more actionable.